tests/cypress/e2e/login_page/login_button.cy.js

```diff
@@ -0,0 +1,19 @@
+import {goToLogin} from '../tools/navigation/nav.js'
+import {login, loginForUser} from '../tools/login/login.js'
+
+describe('Login button test', () => {
+  it('passes', () => {
+    goToLogin();
+
+    // Tests login button type and visibility
+    cy.get('button[class*="green-btn mt-2"]')
+      .should('be.visible')
+      .should('have.attr', 'type', 'submit')
+      .should('have.text', 'Log in');
```

Comment:
This only works for English, do we know in which language the tests would be run? The `_('login')` is a placeholder that is replaced on render with the user language. Using this as a testing validation is a bit tricky.

Reply:
Yes I ran into this! I started the server with language set to non-English and then a test failed. Could we force the tests to run in a certain language?

Reply:
Hm, I think, since you highlighted the tests should be flexible, checking the texts should not be done, because texts are very likely to change.

Reply:
I would suggest to focus on the `id` attribute of most elements. If it is not there, feel free to add it to the templates. The id should be unique by design (if we have duplicate ones this should be fixed). This is unlikely to change and should directly re-present the functionality of the element. For example, the login button could have an `id` attribute `loginButton`.

Reply:
I did exactly this is my rerequest. Maybe I am doing something wrong? Are you still seeing the old files?

Reply:
You did not, but my comment is older than your latest changes. Will review your PR again later today!